frontend/components/Nav/index.tsx

```diff
@@ -1,19 +1,51 @@
 import Link from 'next/link';
 import { StyledNav } from './nav.styles';
 
-const Nav: React.FC = () => {
+const Nav: React.FC<{ userData: { permissionLevel: number; userId: string; username: string } | string }> = ({
```

Comment:
I assume userData prop, in the future would be used in multiple places. So it's better to make a type definition so that it's easier to maintain.

Reply:
Spot on, will create global typings.

Reply:
Done!